docs/index.rst

```diff
@@ -42,71 +42,64 @@ Contents
    :caption: Introduction
    :titlesonly:
 
-   introduction
+   introduction/about-open-ms.md
+   introduction/background.md
+   introduction/how-openms-works.md
 
 .. toctree::
    :maxdepth: 2
    :caption: Getting Started
-   :titlesonly:
 
-   installations/installation-on-gnu-linux
-   installations/installation-on-windows
-   installations/installation-on-macos
-   installations/build-openms-from-source
+   installations/installation-on-gnu-linux.md
+   installations/installation-on-macos.md
+   installations/installation-on-windows.md
 
 .. toctree::
    :maxdepth: 2
-   :caption: Quick Start Guides
-   :titlesonly:
+   :caption: OpenMS Applications and Tools
 
-   guides/user-guides/user-quickstart-guide
-   guides/contributors-quickstart-guide.md
+   openms-applications-and-tools/openms-applications.md
+   openms-applications-and-tools/topp-tools.md
+   openms-applications-and-tools/utils-tools.md
+   openms-applications-and-tools/command-line-interface.md
+   openms-applications-and-tools/visualize-with-openms.md
 
 .. toctree::
    :maxdepth: 2
-   :caption: Tutorials
-   :titlesonly:
+   :caption: Run Workflows with OpenMS Tools
 
-   tutorials/TOPP/TOPP-tutorial
-   tutorials/TOPPAS/TOPPAS-tutorial
-   tutorials/KNIME/KNIME-tutorial
+   run-workflows-with-openms-tools/openms-in-knime.md
+   run-workflows-with-openms-tools/openms-in-nextflow.md
+   run-workflows-with-openms-tools/openms-on-galaxy.md
 
 .. toctree::
    :maxdepth: 2
-   :caption: OpenMS TOPP Tools
-   :titlesonly:
-
-   topp-and-utils/topp-and-utils.md
-   topp-and-utils/adding-new-tool-to-topp.md
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Developer Resources
-   :titlesonly:
-
-   additional-resources/developer-guidelines-for-addding-new-dependent-libraries.md
-   additional-resources/external-code-using-openms.md
-   advanced-resources/custom-compilation.md
-   advanced-resources/build-custom-openms-knime-package.md
+   :caption: Tutorials and Quick Start Guides
+   
+   tutorials-and-quickstart-guides/tutorials.md
+   tutorials-and-quickstart-guides/quickstart-guides.md
 
 .. toctree::
    :maxdepth: 2
-   :caption: OpenMS GitHub Workflow
-   :titlesonly:
+   :caption: Develop with OpenMS
 
-   additional-resources/openms-git-workflow.md
-   additional-resources/reporting-bugs-and-issues.md
-   additional-resources/write-and-label-github-issues.md
-   additional-resources/pull-request-checklist.md
+   develop-with-openms/cplusplus.md
+   develop-with-openms/python.md
```

Comment:
As discussed I would be in favor of renaming "python" to "pyOpenMS". Of course the article needs to mention, that you can use pyOpenMS together with other python modules.

Reply:
Done